### ISSUE TITLE
tcpdump is broken

### ISSUE THREAD

**Issue by Bakaika** [termux/termux-root-packages#38]
```
$ tcpdump
CANNOT LINK EXECUTABLE "tcpdump": library "libcrypto.so.1.0.0" not found
# hangs for a bit...
[1] 8446 abort tcpdump
```
strace:
```
execve("/data/data/com.termux/files/usr/bin/tcpdump", ["tcpdump"], 0x7fd1811fc0 /* 22 vars */) = 0
mmap(NULL, 20480, PROT_NONE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x74da7d0000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7d0000, 20480, "bionic TLS guard page") = 0
mprotect(0x74da7d1000, 12288, PROT_READ|PROT_WRITE) = 0
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7d1000, 12288, "bionic TLS") = 0
set_tid_address(0x74da92e9d8)           = 22597
faccessat(AT_FDCWD, "/dev/urandom", R_OK) = 0
futex(0x74da92b280, FUTEX_WAKE_PRIVATE, 2147483647) = 0
getrandom("\x7c\x50\x02\x5b\x81\x50\x21\x1a\x69\xd0\x65\x97\xb8\xbc\x48\xa0\xc5\x7e\x2e\x88\xce\x68\xba\x8d\x6c\xc4\x11\x79\x8c\x22\xaa\x0b"..., 40, GRND_NONBLOCK) = 40
mmap(NULL, 1104, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x74da7cf000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7cf000, 1104, "arc4random data") = 0
mmap(NULL, 20480, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x74da7ca000
mprotect(0x74da7ca000, 4096, PROT_NONE) = 0
sigaltstack({ss_sp=0x74da7cb000, ss_flags=0, ss_size=16384}, NULL) = 0
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7cb000, 16384, "thread signal stack") = 0
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7ca000, 4096, "thread signal stack guard page") = 0
mprotect(0x74da8bc000, 16384, PROT_READ) = 0
mprotect(0x74da92d000, 4096, PROT_READ) = 0
mprotect(0x74da92d000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da92d000, 4096, PROT_READ) = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x74da7c9000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c9000, 4096, "atexit handlers") = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c8000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c8000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 24, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c7000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c7000, 24, "linker_alloc_vector") = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c6000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c6000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 24, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c5000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c5000, 24, "linker_alloc_vector") = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c4000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c4000, 4096, "linker_alloc") = 0
mprotect(0x74da7c4000, 4096, PROT_READ|PROT_WRITE) = 0
newfstatat(AT_FDCWD, "/dev/__properties__", {st_mode=S_IFDIR|0711, st_size=1800, ...}, 0) = 0
openat(AT_FDCWD, "/property_contexts", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
faccessat(AT_FDCWD, "/system/etc/selinux/plat_property_contexts", R_OK) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/plat_property_contexts", O_RDONLY|O_CLOEXEC) = 3
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c3000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c3000, 4096, "linker_alloc_lob") = 0
fstat(3, {st_mode=S_IFREG|0644, st_size=4323, ...}) = 0
mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c1000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c1000, 8192, "linker_alloc_lob") = 0
read(3, "asan.restore_reboot  u:object_r:"..., 4096) = 4096
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c0000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c0000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 24, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7bf000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7bf000, 24, "linker_alloc_vector") = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7be000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7be000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 24, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7bd000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7bd000, 24, "linker_alloc_vector") = 0
read(3, " u:object_r:system_radio_prop:s0"..., 4096) = 227
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7bc000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7bc000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 48, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7bb000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7bb000, 48, "linker_alloc_vector") = 0
munmap(0x74da7c7000, 24)                = 0
read(3, "", 4096)                       = 0
munmap(0x74da7c0000, 4096)              = 0
close(3)                                = 0
munmap(0x74da7c1000, 8192)              = 0
openat(AT_FDCWD, "/nonplat_property_contexts", O_RDONLY|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0644, st_size=7165, ...}) = 0
mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c1000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c1000, 8192, "linker_alloc_lob") = 0
read(3, "audioflinger.                   "..., 4096) = 4096
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c7000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c7000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c0000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c0000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 48, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7ba000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7ba000, 48, "linker_alloc_vector") = 0
munmap(0x74da7bd000, 24)                = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7bd000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7bd000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 96, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7b9000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7b9000, 96, "linker_alloc_vector") = 0
munmap(0x74da7bb000, 48)                = 0
read(3, "    u:object_r:recovery_prop:s0\n"..., 4096) = 3069
read(3, "", 4096)                       = 0
munmap(0x74da7c7000, 4096)              = 0
close(3)                                = 0
munmap(0x74da7c1000, 8192)              = 0
openat(AT_FDCWD, "/dev/__properties__/properties_serial", O_RDONLY|O_NOFOLLOW|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0444, st_size=196608, ...}) = 0
mmap(NULL, 196608, PROT_READ, MAP_SHARED, 3, 0) = 0x74da789000
close(3)                                = 0
mmap(NULL, 12288, PROT_NONE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x74da786000
mprotect(0x74da787000, 4096, PROT_READ|PROT_WRITE) = 0
rt_sigaction(SIGABRT, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGBUS, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGFPE, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGILL, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGSEGV, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGSTKFLT, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGSYS, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGTRAP, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_3, {sa_handler=0x74da7fe104, sa_mask=~[], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
prctl(PR_GET_DUMPABLE)                  = 1 (SUID_DUMP_USER)
futex(0x74da92b280, FUTEX_WAKE_PRIVATE, 2147483647) = 0
openat(AT_FDCWD, "/dev/__properties__/u:object_r:debug_prop:s0", O_RDONLY|O_NOFOLLOW|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0444, st_size=196608, ...}) = 0
mmap(NULL, 196608, PROT_READ, MAP_SHARED, 3, 0) = 0x74da756000
close(3)                                = 0
futex(0x74da92b280, FUTEX_WAKE_PRIVATE, 2147483647) = 0
newfstatat(AT_FDCWD, "/proc/self/exe", {st_mode=S_IFREG|0700, st_size=1065080, ...}, 0) = 0
mprotect(0x74da7c9000, 4096, PROT_READ|PROT_WRITE) = 0
mprotect(0x74da7c9000, 4096, PROT_READ) = 0
futex(0x74da92b280, FUTEX_WAKE_PRIVATE, 2147483647) = 0
readlinkat(AT_FDCWD, "/proc/self/exe", "/data/data/com.termux/files/usr/"..., 4096) = 43
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c7000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c7000, 4096, "linker_alloc") = 0
newfstatat(AT_FDCWD, "/data", {st_mode=S_IFDIR|0771, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/data/data", {st_mode=S_IFDIR|0771, st_size=12288, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/data/data/com.termux", {st_mode=S_IFDIR|0700, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/data/data/com.termux/files", {st_mode=S_IFDIR|0700, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/data/data/com.termux/files/usr", {st_mode=S_IFDIR|0700, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/data/data/com.termux/files/usr/lib", {st_mode=S_IFDIR|0700, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/data/data/com.termux/files/usr/lib", {st_mode=S_IFDIR|0700, st_size=4096, ...}, 0) = 0
openat(AT_FDCWD, "/system/etc/ld.config.txt", O_RDONLY|O_NOFOLLOW|O_CLOEXEC) = 3
fstat(3, {st_mode=S_IFREG|0644, st_size=486, ...}) = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c2000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c2000, 4096, "linker_alloc_small_objects") = 0
mmap(NULL, 24, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c1000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c1000, 24, "linker_alloc_vector") = 0
read(3, "# Copyright (C) 2017 The Android"..., 1024) = 486
read(3, "", 1024)                       = 0
close(3)                                = 0
munmap(0x74da7c2000, 4096)              = 0
newfstatat(AT_FDCWD, "/system", {st_mode=S_IFDIR|0755, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/system/lib64", {st_mode=S_IFDIR|0755, st_size=20480, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/vendor", {st_mode=S_IFLNK|0777, st_size=14, ...}, AT_SYMLINK_NOFOLLOW) = 0
readlinkat(AT_FDCWD, "/vendor", "/system/vendor", 4095) = 14
newfstatat(AT_FDCWD, "/system", {st_mode=S_IFDIR|0755, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/system/vendor", {st_mode=S_IFDIR|0755, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
newfstatat(AT_FDCWD, "/system/vendor/lib64", {st_mode=S_IFDIR|0755, st_size=4096, ...}, AT_SYMLINK_NOFOLLOW) = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7c2000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7c2000, 4096, "linker_alloc") = 0
openat(AT_FDCWD, "/data/data/com.termux/files/usr/lib/libtermux-exec.so", O_RDONLY|O_CLOEXEC) = 3
mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da754000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da754000, 8192, "linker_alloc_lob") = 0
mmap(NULL, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da752000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da752000, 8192, "linker_alloc_lob") = 0
readlinkat(AT_FDCWD, "/proc/self/fd/3", "/data/data/com.termux/files/usr/"..., 4096) = 53
munmap(0x74da752000, 8192)              = 0
munmap(0x74da754000, 8192)              = 0
fstat(3, {st_mode=S_IFREG|0600, st_size=5720, ...}) = 0
fstatfs(3, {f_type=EXT2_SUPER_MAGIC, f_bsize=4096, f_blocks=6340372, f_bfree=4921096, f_bavail=4921096, f_files=1618848, f_ffree=1603337, f_fsid={val=[3546324968, 77008235]}, f_namelen=255, f_frsize=4096, f_flags=ST_VALID|ST_NOSUID|ST_NODEV|ST_NOATIME}) = 0
mmap(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, 0, 0) = 0x74da7bb000
prctl(PR_SET_VMA, PR_SET_VMA_ANON_NAME, 0x74da7bb000, 4096, "linker_alloc_small_objects") = 0
pread64(3, "\177ELF\2\1\1\0\0\0\0\0\0\0\0\0\3\0\267\0\1\0\0\0\0\0\0\0\0\0\0\0"..., 64, 0) = 64
mmap(NULL, 464, PROT_READ, MAP_PRIVATE, 3, 0) = 0x74da755000
mmap(NULL, 2032, PROT_READ, MAP_PRIVATE, 3, 0x1000) = 0x74da754000
mmap(NULL, 7728, PROT_READ, MAP_PRIVATE, 3, 0) = 0x74da752000
mmap(NULL, 1813, PROT_READ, MAP_PRIVATE, 3, 0) = 0x74da751000
openat(AT_FDCWD, "/data/data/com.termux/files/usr/lib/libcrypto.so.1.0.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/system/lib64/libcrypto.so.1.0.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/system/vendor/lib64/libcrypto.so.1.0.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
openat(AT_FDCWD, "/dev/__properties__/u:object_r:default_prop:s0", O_RDONLY|O_NOFOLLOW|O_CLOEXEC) = 4
fstat(4, {st_mode=S_IFREG|0444, st_size=196608, ...}) = 0
mmap(NULL, 196608, PROT_READ, MAP_SHARED, 4, 0) = 0x74da721000
close(4)                                = 0
close(3)                                = 0
munmap(0x74da751000, 1813)              = 0
munmap(0x74da752000, 7728)              = 0
munmap(0x74da754000, 2032)              = 0
munmap(0x74da755000, 464)               = 0
munmap(0x74da7bb000, 4096)              = 0
writev(2, [{iov_base="CANNOT LINK EXECUTABLE \"tcpdump\""..., iov_len=72}, {iov_base="\n", iov_len=1}], 2CANNOT LINK EXECUTABLE "tcpdump": library "libcrypto.so.1.0.0" not found
) = 73
socket(AF_UNIX, SOCK_DGRAM|SOCK_CLOEXEC|SOCK_NONBLOCK, 0) = 3
openat(AT_FDCWD, "/dev/__properties__/u:object_r:system_prop:s0", O_RDONLY|O_NOFOLLOW|O_CLOEXEC) = 4
fstat(4, {st_mode=S_IFREG|0444, st_size=196608, ...}) = 0
mmap(NULL, 196608, PROT_READ, MAP_SHARED, 4, 0) = 0x74da6f1000
close(4)                                = 0
connect(3, {sa_family=AF_UNIX, sun_path="/dev/socket/logdw"}, 110) = 0
openat(AT_FDCWD, "/dev/__properties__/u:object_r:logd_prop:s0", O_RDONLY|O_NOFOLLOW|O_CLOEXEC) = 4
fstat(4, {st_mode=S_IFREG|0444, st_size=196608, ...}) = 0
mmap(NULL, 196608, PROT_READ, MAP_SHARED, 4, 0) = 0x74da6c1000
close(4)                                = 0
writev(3, [{iov_base="\4", iov_len=1}, {iov_base="EX", iov_len=2}, {iov_base="C\17\257[3/\3268", iov_len=8}, {iov_base="\7", iov_len=1}, {iov_base="libc\0", iov_len=5}, {iov_base="CANNOT LINK EXECUTABLE \"tcpdump\""..., iov_len=73}], 6) = 90
close(3)                                = 0
mmap(NULL, 81, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x74da7bb000
getpid()                                = 22597
gettid()                                = 22597
rt_sigprocmask(SIG_SETMASK, ~[ABRT], [], 8) = 0
tgkill(22597, 22597, SIGABRT)           = 0
--- SIGABRT {si_signo=SIGABRT, si_code=SI_TKILL, si_pid=22597, si_uid=10177} ---
rt_sigaction(SIGABRT, NULL, {sa_handler=0x74da7fe104, sa_mask=~[KILL STOP], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, 8) = 0
prctl(PR_GET_NO_NEW_PRIVS, 0, 0, 0, 0)  = 0
prctl(PR_GET_NAME, "tcpdump")           = 0
gettid()                                = 22597
socket(AF_UNIX, SOCK_DGRAM|SOCK_CLOEXEC|SOCK_NONBLOCK, 0) = 3
connect(3, {sa_family=AF_UNIX, sun_path="/dev/socket/logdw"}, 110) = 0
writev(3, [{iov_base="\4", iov_len=1}, {iov_base="EX", iov_len=2}, {iov_base="C\17\257[\255\316\3558", iov_len=8}, {iov_base="\7", iov_len=1}, {iov_base="libc\0", iov_len=5}, {iov_base="Fatal signal 6 (SIGABRT), code -"..., iov_len=57}], 6) = 74
close(3)                                = 0
gettid()                                = 22597
prctl(PR_SET_DUMPABLE, SUID_DUMP_USER)  = 0
clone(child_stack=0x74da787fe0, flags=CLONE_VM|CLONE_SIGHAND|CLONE_THREAD|CLONE_CHILD_CLEARTID|CLONE_CHILD_SETTID, child_tidptr=0x74da7cdd08) = 22616
futex(0x74da7cdd08, FUTEX_WAIT, 4294967295, NULL) = -1 EAGAIN (Try again)
futex(0x74da7cdd08, FUTEX_WAIT, 22616, NULL) = 0
rt_sigaction(SIGABRT, {sa_handler=SIG_DFL, sa_mask=[], sa_flags=SA_RESTART}, {sa_handler=0x74da7fe104, sa_mask=~[KILL STOP], sa_flags=SA_ONSTACK|SA_RESTART|SA_SIGINFO}, 8) = 0
rt_sigaction(SIGABRT, {sa_handler=SIG_DFL, sa_mask=[], sa_flags=SA_RESTART}, {sa_handler=SIG_DFL, sa_mask=[], sa_flags=SA_RESTART}, 8) = 0
getpid()                                = 22597
gettid()                                = 22597
rt_tgsigqueueinfo(22597, 22597, SIGABRT, {si_signo=SIGABRT, si_code=SI_TKILL, si_pid=22597, si_uid=10177}) = 0
rt_sigreturn({mask=~[ABRT KILL STOP]})  = 0
--- SIGABRT {si_signo=SIGABRT, si_code=SI_TKILL, si_pid=22597, si_uid=10177} ---
+++ killed by SIGABRT +++
```
termux-info:
```
Updatable packages:
All packages up to date
System information:
Linux localhost 4.4.154-MCK-O-v2 #1 SMP PREEMPT Sat Sep 22 19:23:37 CEST 2018 aarch64 Android
Termux-packages arch:
aarch64
Android version:
8.0.0
Device manufacturer:
samsung
Device model:
SM-A530F
```

**Comment by fornwall** [comment]
Run `pkg up` to upgrade installed packages - this should fix the problem!

**Comment by Grimler91** [comment]
Yepp, fixed in a1cd4e5dc7a97e89fc47a7e08f24a5ba2281ada2, updated tcpdump is available from the repo. Thanks for reporting!
